src/main.rs

```diff
@@ -74,15 +89,18 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let cargo_toml_path = validate_workspace_path(cmd.path)?;
 
     // Decide which branch data to use based on the branch name
-    let crates_versions: BTreeMap<String, String> =
+    let mut crates_versions: BTreeMap<String, String> =
         get_version_mapping_with_fallback(DEFAULT_GIT_SERVER, &version).await?;
 
-    update_dependencies(
-        &cargo_toml_path,
-        &crates_versions,
-        cmd.overwrite,
-        cmd.check,
-    )?;
+    let orml_crates_version = if cmd.orml {
+        get_orml_crates_and_version(DEFAULT_GIT_SERVER, &version).await?
+    } else {
+        None
+    };
+
+    include_orml_crates_in_version_mapping(&mut crates_versions, orml_crates_version);
```

Comment:
Only call this if `cmd.orml` is set?

Reply:
Changed it to get called only when `cmd.orml` is set. No more needless function call and empty returns.